### ISSUE TITLE
about performance

### ISSUE THREAD

**Issue by Terminal-K** [PRBonn/MapMOS#16]
hi, sorry to bother you again.
I would like to understand why MapMOS performs well on the validation (with the iou of 86.1%) of SemanticKITTI but experiences a significant drop in performance on the test (with the iou of 66.0%) as reported in the paper. From my understanding, other methods do not exhibit such a large discrepancy. Could this difference be attributed to KISS-ICP?

**Comment by benemer** [comment]
This is true for most approaches; see Table I of our paper. MapMOS is just working a lot better on the validation set, making this gap more evident.

Since the test labels are not public, analyzing failure cases is impossible. But here are two possible reasons in my view:

First, we use the validation set to pick the final model while training. This means there is already some bias because you select the weights that perform best on the validation set.

Second, you compare the result on a single sequence in an urban environment to the result on multiple sequences with more diverse environments and driving situations, such as highway driving. So the two sets of sequences are most likely not the same in terms of complexity.

**Comment by Terminal-K** [comment]
Thanks for your insightful response, which answers my question and clears up my confusion!
Wishing you all the best!
